docs/setup/upgrade/logging-configuration-changes.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[logging-configuration-changes]]
+=== Logging configuration changes
+
+WARNING: {kib} 8.0 and later uses a logging system. Be sure to read the documentation for your version of {kib} before proceeding.
+
+[[logging-pattern-format-old-and-new-example]]
+[options="header"]
+|===
+
+| Parameter | Platform log record in **pattern** format | Legacy Platform log record **tedocs/xt** format
```

Comment:
```suggestion
| Parameter | Log record in **pattern** format | Legacy log record in **text** format
```

Removed "platform" since end-users don't know or need to know what it is.